src/Templates/src/maui-contentpage/NewPage1.xaml.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Xaml;
+
+namespace MauiApp1
```

Comment:
Do we maybe need to add a parameter or something to the json for the namespace? I saw this in ios for a new view controller:

```json
{
    "symbols": {
      "namespace": {
        "description": "namespace for the generated code",
        "replaces": "iOSApp1",
        "type": "parameter"
      }
    }
}
```